lib/Plerd/Post.pm

```diff
@@ -130,9 +130,13 @@ sub _build_published_filename {
 sub _build_uri {
     my $self = shift;
 
+    my $base_uri = $self->plerd->base_uri;
+    if ($base_uri =~ m,[^/]$,) {
```

Comment:
Could you please change the regex syntax here to match those of regexes elsewhere in the code? (See e.g. various lines in the _process_source_file() method of Post.pm)